pkgs/build-support/rust/fetch-cargo-vendor-util.py

```diff
@@ -119,7 +120,7 @@ def create_vendor_staging(lockfile_path: Path, out_dir: Path) -> None:
     shutil.copy(lockfile_path, out_dir / "Cargo.lock")
 
     # create a pool with at most 10 concurrent jobs
-    with mp.Pool(min(10, mp.cpu_count())) as pool:
+    with mp.Pool(min(1, mp.cpu_count())) as pool:
```

Comment:
By the way I think you should tone this down. I got an error a while ago saying connection limits have exceeded. Happened only once though.

Reply:
Currently changed it to have the logs in order. Will consider